fabric-registry-sync-v0/src/main/resources/quilted_fabric_registry_sync_v0.mixins.json

```diff
@@ -1,19 +1,22 @@
 {
   "required": true,
   "package": "net.fabricmc.fabric.mixin.registry.sync",
-  "compatibilityLevel": "JAVA_17",
+  "compatibilityLevel": "JAVA_16",
```

Comment:
nonononono
```suggestion
  "compatibilityLevel": "JAVA_17",
```

Reply:
I have no clue how that got changed what the hell